eks-without-vpc/readme.md

```diff
@@ -0,0 +1,26 @@
+# EKS without VPC
```

Comment:
Filename should be `README.md` (capitalized)

Reply:
changed